plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -43,15 +50,39 @@ public class IcebergTableProperties
     public static final String SORTED_BY_PROPERTY = "sorted_by";
     public static final String LOCATION_PROPERTY = "location";
     public static final String FORMAT_VERSION_PROPERTY = "format_version";
-    public static final String ORC_BLOOM_FILTER_COLUMNS_PROPERTY = "orc_bloom_filter_columns";
     public static final String ORC_BLOOM_FILTER_FPP_PROPERTY = "orc_bloom_filter_fpp";
+    public static final String ORC_BLOOM_FILTER_COLUMNS_PROPERTY = "orc_bloom_filter_columns";
+    public static final String EXTRA_PROPERTIES_PROPERTY = "extra_properties";
+
+    public static final String INTERNAL_TABLE_COMMENT_PROPERTY = "comment";
+
+    // comment is not a supported property to be defined through the WITH clause; they are defined through the COMMENT clause.
+    public static final ImmutableSet<String> SUPPORTED_PROPERTIES = ImmutableSet.<String>builder()
```

Comment:
```suggestion
    public static final Set<String> SUPPORTED_PROPERTIES = ImmutableSet.<String>builder()
```
Same for PROTECTED_ICEBERG_NATIVE_PROPERTIES